options.c

```diff
@@ -128,7 +128,9 @@ void parse_args(state *st, int argc, char *argv[])
 			case 'D': sstrlcpy(st->server_description, optarg); break;
 			case 'L': sstrlcpy(st->server_location, optarg); break;
 			case 'A': sstrlcpy(st->server_admin, optarg); break;
-
+#ifdef __OpenBSD__
```

Comment:
There should probably be some kind of check here, so that if it isn't obsd, it errors out.

Reply:
Good point. I've made the addition of `-U` compile-time conditional.